python/CMakeLists.txt

```diff
@@ -18,9 +19,8 @@ pybind11_add_module(cpp SHARED
    dolfinx_mpc/mpc.cpp
    dolfinx_mpc/dolfinx_mpc.cpp)
 
-target_link_libraries(cpp PRIVATE pybind11::module dolfinx_mpc)
-target_link_libraries(cpp PUBLIC dolfinx)
-target_include_directories(cpp PUBLIC dolfinx)
+target_link_libraries(cpp PRIVATE pybind11::module)
+target_link_libraries(cpp PUBLIC dolfinx_mpc)
```

Comment:
dolfinx_mpc moved from private to public - this pulls in dolfinx, etc., which are dolfinx_mpc's own public dependencies.